### PR TITLE
 CIFAR-10 benchmark. Benchmarks refactoring

### PR DESCRIPTION
**Pull request by apertovs**

### CODE REVIEW

**Comment by TolyaTalamanov** [comment]
@apertovs re-run CI

File: benchmarks/utils/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+add_library(
+    benchmark_output_utils STATIC
+    output_utils.cpp)
+target_include_directories(benchmark_output_utils PUBLIC .)
```
Comment (TolyaTalamanov) [inline benchmarks/utils/CMakeLists.txt:4]: Can it be `PRIVATE` ?

Comment (TolyaTalamanov) [reply benchmarks/utils/CMakeLists.txt:4]: Not sure that understand `.` in that case, could you specify which libraries are used

**Review by TolyaTalamanov — APPROVED** [review]

**Comment by TolyaTalamanov** [comment]
Can we add this to CI ?

File: benchmarks/utils/CMakeLists.txt (hunk 1 above)
Comment (apertovs) [reply benchmarks/utils/CMakeLists.txt:4]: It provides access to output_utils.hpp

**Comment by apertovs** [comment]
> Can we add this to CI ?

Yes, I did it in last commit.

**Comment by TolyaTalamanov** [comment]
Restart CI

**Comment by TolyaTalamanov** [comment]
Looks like we must decrease number of epochs due time limit

**Comment by TolyaTalamanov** [comment]
Great! Thanks 👍

**Pull request merged**